### DISCUSSION TITLE
Add "add_post_type_support" options when extending builtin post types

### DISCUSSION THREAD

**Discussion by stefanobartoletti** [pods-framework/pods#6783] [Ideas]
Would it be possible to add "add_post_type_support" options when extending builtin post types?

Just as an example, to add the Excerpt to Pages.

I see that a "Supports" section is available under the "Advanced options" tab when creating a custom post type.

Would it be possible to add this even when extending builtin post types?

**Comment by quasel** [comment]
Never thought about that, but as the CPT is already registered but maybe I'm overthinking that and there is an easy and quick filter @sc0ttkclark ?
